### DISCUSSION TITLE
Add Dynamic Split Keybind Based on Active Split Dimensions

### DISCUSSION THREAD

**Discussion by BBennettChiba** [ghostty-org/ghostty#3706] [Feature Requests, Ideas] [answered]
Hello,

I would like to request a feature or ask if it's possible to achieve the following behavior through configuration:

In my current terminal setup (wezterm), I have a keybind that dynamically splits the active pane/split within the terminal window based on its dimensions:

Horizontal split: If the active split is taller than it is wide.
Vertical split: If the active split is wider than it is tall.
This dynamic behavior adapts to the dimensions of the currently focused split, making it easier to manage space efficiently within the terminal.

Is there a way to achieve this in Ghostty? If not, could this feature be considered for future implementation?

Thank you for your time and consideration!

**Comment by pluiedev** [comment, marked as answer]
[`new_split:auto`](https://ghostty.org/docs/config/keybind/reference#new_split) does exactly what you want

**Reply by BBennettChiba** [reply]
thank you
